paywall/src/__tests__/unlock.js/setupPostOffices.test.ts

```diff
@@ -142,27 +142,6 @@ describe('setupPostOffice', () => {
     expect(fakeUIIframe.className).toBe('unlock start show')
   })
 
```

Comment:
Is the removal of this test intentional?

Reply:
it was moved to web3Proxy.test.ts